docs/CHANGELOG.asciidoc

```diff
@@ -28,6 +28,12 @@
 
 //=== Regressions
 
+== {es} version 8.9.0
+
+=== Enhancements
+
+* Retrieve real memory usage of PyTorch inference (See {ml-pull}2487[#2487].)
```

Comment:
On it's own this isn't a user-facing change. There will need to be a Java PR to make this do anything as far as end users are concerned. So I would add the release note for the overall functionality in the Java PR. The C++ release notes have to be manually copied into the overall release notes, whereas the Java release notes are auto-generated, so less work.

Reply:
I made this entry, because of the github workflow check. I happily revert and ignore the gh check. 

Maybe it can be documented in the process that this check isn't mandatory to pass.

Reply:
If you change `>enhancement` to `>non-issue` then a release note won't be expected. Then for the Java PR set the labels the other way around.